pr_inspector/tools/checklist/tool.py

```
@@ -0,0 +1,111 @@
+              "schema": output_schema
+          }
+      }
+    )
+    content: str = response.choices[0].message.content
+    json_data = json.loads(content)
+    return ChecklistOutput(**json_data)
+
+
+def transform_response_to_markdown(response: ChecklistOutput) -> str:
+    markdown = f"# Checklist for PR: {pr_url}\n\n"
+    markdown += f"## Key Files & Review Order\n\n"
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)